### PR TITLE
feat: flagged exception autocapture opt in

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

We can't release this widely until we've resolved or worked around ingestion burden it adds.

But we can test it so that when they're solved we're ready to go.

## Changes

* adds the team field
* and setting to toggle it (hidden behind a flag)

### autocaptured exception

Screenshot 2023-06-07 at 14 34 26

### toggle

Screenshot 2023-06-07 at 14 34 58

## How did you test this code?

dev tests and 👀  locally

### CODE REVIEW

File: frontend/src/scenes/project/Settings/AutocaptureSettings.tsx [hunk 1]
```diff
@@ -17,19 +20,40 @@ export function AutocaptureSettings(): JSX.Element {
...
-                        autocapture_opt_out: !checked,
-                    })
-                    reportIngestionAutocaptureToggled(!checked)
-                }}
-                checked={!currentTeam?.autocapture_opt_out}
-                disabled={userLoading}
-                label="Enable Autocapture"
-                bordered
-            />
+            <div className={'flex flex-col space-y-2'}>
```
Comment (benjackwhite) [inline frontend/src/scenes/project/Settings/AutocaptureSettings.tsx:23]: flex and flex-col are probably not needed here. If they are then `gap-2` is preferable for flexed content

```suggestion
            <div className="space-y-2">
```

**Review by benjackwhite — APPROVED** [review]

**Comment by pauldambra** [comment]
@neilkakkar this adds a new item in the decide response... I don't _think_ I've done that in a silly way but...

File: posthog/api/decide.py [hunk 2]
```diff
@@ -185,6 +185,7 @@ def get_decide(request: HttpRequest):
 
             response["capturePerformance"] = True if team.capture_performance_opt_in else False
             response["autocapture_opt_out"] = True if team.autocapture_opt_out else False
+            response["autocaptureExceptions"] = True if team.autocapture_exceptions_opt_in else False
```
Comment (neilkakkar) [inline posthog/api/decide.py:188]: almost! See below on Line 210 - just add to that test, so no one in the future can bork this - this looks fine :)

**Review by neilkakkar — APPROVED** [review]
Thanks for tagging!

Just add to `test_decide_doesnt_error_out_when_database_is_down` and you should be good to go.

**Pull request merged**